.github/workflows/build.yaml

```
@@ -21,6 +21,13 @@ jobs:
           secret: ADH_PIPELINE_PAT
           repo: vpaza/gitops
 
+        facility: [zlc]
+        include:
+        - facility: zlc
+          name: KZLC
+          secret: ADH_PIPELINE_PAT
```

```suggestion
          secret: KZLC_GITOPS_TOKEN
```

ZLCs secret is KZLC_GITOPS_TOKEN